### PR TITLE
Update for `BackgroundFetchEvent` & `BackgroundFetchUpdateUIEvent`

### PR DESCRIPTION
**Pull request by skyclouds2001**
### Description

update inherit info for `BackgroundFetchEvent` and `BackgroundFetchUpdateUIEvent`

add missing syntax list for `BackgroundFetchUpdateUIEvent.updateUI()`

some other update

### CODE REVIEW

File: files/en-us/web/api/backgroundfetchevent/registration/index.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ A {{domxref("BackgroundFetchRegistration")}}.
 
 ## Examples
 
-In this example, if the user clicks on the user interface displaying the download progress, this fires the `onbackgroundfetchclick` event. The current {{domxref("BackgroundFetchRegistration")}} is returned by calling `event.registration`.
+In this example, if the user clicks on the user interface displaying the download progress, this fires the `backgroundfetchclick` event. The current {{domxref("BackgroundFetchRegistration")}} is returned by calling `event.registration`.
```
Comment (bsmth) [inline files/en-us/web/api/backgroundfetchevent/registration/index.md:21]:
```suggestion
In this example, if the user clicks on the user interface displaying the download progress, this fires the [`backgroundfetchclick` event](/en-US/docs/Web/API/ServiceWorkerGlobalScope/backgroundfetchclick_event). The current {{domxref("BackgroundFetchRegistration")}} is returned by calling `event.registration`.
```

File: files/en-us/web/api/backgroundfetchevent/index.md [hunk 2]
```diff
@@ -9,9 +9,9 @@ browser-compat: api.BackgroundFetchEvent
 
 {{APIRef("Background Fetch API")}}{{SeeCompatTable}}
 
-The **`BackgroundFetchEvent`** interface of the {{domxref('Background Fetch API','','',' ')}} is the event type for background fetch events dispatched on the {{domxref("ServiceWorkerGlobalScope", "service worker global scope")}}.
+The **`BackgroundFetchEvent`** interface of the {{domxref('Background Fetch API','','',' ')}} is the event type for background fetch events dispatched on the {{domxref("ServiceWorkerGlobalScope", "service worker global scope", "", "nocode")}}.
 
-It is the event type passed to `onbackgroundfetchabort` and `onbackgroundfetchclick`.
+It is the event type passed to `backgroundfetchabort` and `backgroundfetchclick`.
```
Comment (bsmth) [inline files/en-us/web/api/backgroundfetchevent/index.md:14]: Commented below, should we be linking to https://developer.mozilla.org/en-US/docs/Web/API/ServiceWorkerGlobalScope/backgroundfetchclick_event and https://developer.mozilla.org/en-US/docs/Web/API/ServiceWorkerGlobalScope/backgroundfetchabort_event ?

Comment (skyclouds2001) [inline files/en-us/web/api/backgroundfetchevent/index.md:14]:
```suggestion
It is the event type passed to {{domxref("ServiceWorkerGlobalScope/backgroundfetchclick_event", "backgroundfetchclick"}} event and {{domxref("ServiceWorkerGlobalScope/backgroundfetchabort_event", "backgroundfetchabort")}} event.
```

File: files/en-us/web/api/backgroundfetchevent/registration/index.md (hunk 1 above)
Comment (skyclouds2001) [inline files/en-us/web/api/backgroundfetchevent/registration/index.md:21]:
```suggestion
In this example, if the user clicks on the user interface displaying the download progress, this fires the {{domxref("ServiceWorkerGlobalScope/backgroundfetchabort_event", "backgroundfetchabort")}} event. The current {{domxref("BackgroundFetchRegistration")}} is returned by calling `event.registration`.
```

File: files/en-us/web/api/backgroundfetchevent/index.md [hunk 3]
```diff
@@ -9,9 +9,9 @@ browser-compat: api.BackgroundFetchEvent
 
 {{APIRef("Background Fetch API")}}{{SeeCompatTable}}
 
-The **`BackgroundFetchEvent`** interface of the {{domxref('Background Fetch API','','',' ')}} is the event type for background fetch events dispatched on the {{domxref("ServiceWorkerGlobalScope", "service worker global scope")}}.
+The **`BackgroundFetchEvent`** interface of the {{domxref('Background Fetch API','','',' ')}} is the event type for background fetch events dispatched on the {{domxref("ServiceWorkerGlobalScope", "service worker global scope", "", "nocode")}}.
 
-It is the event type passed to `onbackgroundfetchabort` and `onbackgroundfetchclick`.
+It is the event type passed to {{domxref("ServiceWorkerGlobalScope/backgroundfetchclick_event", "backgroundfetchclick"}} event and {{domxref("ServiceWorkerGlobalScope/backgroundfetchabort_event", "backgroundfetchabort")}} event.
```
Comment (skyclouds2001) [inline files/en-us/web/api/backgroundfetchevent/index.md:14]:
```suggestion
It is the event type passed to {{domxref("ServiceWorkerGlobalScope/backgroundfetchclick_event", "backgroundfetchclick")}} event and {{domxref("ServiceWorkerGlobalScope/backgroundfetchabort_event", "backgroundfetchabort")}} event.
```

File: files/en-us/web/api/backgroundfetchevent/registration/index.md [hunk 4]
```diff
@@ -18,7 +18,7 @@ A {{domxref("BackgroundFetchRegistration")}}.
 
 ## Examples
 
-In this example, if the user clicks on the user interface displaying the download progress, this fires the `onbackgroundfetchclick` event. The current {{domxref("BackgroundFetchRegistration")}} is returned by calling `event.registration`.
+In this example, if the user clicks on the user interface displaying the download progress, this fires the {{domxref("ServiceWorkerGlobalScope/backgroundfetchabort_event", "backgroundfetchabort")}} event. The current {{domxref("BackgroundFetchRegistration")}} is returned by calling `event.registration`.
```
Comment (skyclouds2001) [inline files/en-us/web/api/backgroundfetchevent/registration/index.md:21]:
```suggestion
In this example, if the user clicks on the user interface displaying the download progress, this fires the {{domxref("ServiceWorkerGlobalScope/backgroundfetchclick_event", "backgroundfetchclick")}} event. The current {{domxref("BackgroundFetchRegistration")}} is returned by calling `event.registration`.
```

File: files/en-us/web/api/backgroundfetchevent/index.md [hunk 5]
```diff
@@ -9,9 +9,9 @@ browser-compat: api.BackgroundFetchEvent
 
 {{APIRef("Background Fetch API")}}{{SeeCompatTable}}
 
-The **`BackgroundFetchEvent`** interface of the {{domxref('Background Fetch API','','',' ')}} is the event type for background fetch events dispatched on the {{domxref("ServiceWorkerGlobalScope", "service worker global scope")}}.
+The **`BackgroundFetchEvent`** interface of the {{domxref('Background Fetch API','','',' ')}} is the event type for background fetch events dispatched on the {{domxref("ServiceWorkerGlobalScope", "service worker global scope", "", "nocode")}}.
```
Comment (bsmth) [inline files/en-us/web/api/backgroundfetchevent/index.md:12]: I think this format is easier to understand, let's do the same:

```suggestion
The **`BackgroundFetchEvent`** interface of the {{domxref('Background Fetch API', "", "", "nocode")}} is the event type for background fetch events dispatched on the {{domxref("ServiceWorkerGlobalScope", "service worker global scope", "", "nocode")}}.
```

**Review by bsmth — APPROVED** [review]
Great, thank you!

**Pull request merged**